llvm/lib/Transforms/PeepholeOptimizationCourse/ReplaseAddNewPassManager/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_llvm_component_library(
+	LLVMReplaseAddNewPassManager
```

Comment:
Do no see where you are linking against that library. Did you run pass through new pass manager?

Reply:
I fix it